mail_create_partner/static/src/js/mail_create_partner.js

```diff
@@ -31,14 +32,15 @@ odoo.define('mail_create_partner.create_partner_object', function (require) {
                 view_type: 'form',
                 views: [[false, 'form']],
                 target: 'new',
-                context: {'default_message_id': message_id}
+                context: {'default_message_id': message_id},
             };
+            this.message_id = message_id;
 
-            self.do_action(action, {
+            this.do_action(action, {
                 'on_close': function(){
-                    var message = base_obj.chat_manager.get_message(self.message_id);
-                    chat_manager.bus.trigger('update_message', message);
-                    self.fetch_and_render_thread();
+//                    var message = base_obj.chat_manager.get_message(this.message_id);
```

Comment:
why not delete?

Reply:
It acts as a reminder.

Reply:
No. It's nothing. It's just I didn't want to delete. May serve as hint if in the future needs an improvement.